stockflux-securities-master/src/components/form-view/form-controls/text-field/TextField.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './TextField.css';
+
+const TextField = ({ label, id, value, disabled, onChange }) => {
+  return (
+    <span className="input">
+      <label htmlFor={id}>{label}</label>
+      <input
+        className={disabled}
+        id={id}
+        disabled={disabled}
+        value={value}
+        onChange={onChange}
+      />
+    </span>
+  );
+};
+
+TextField.propTypes = {
+  label: PropTypes.string.isRequired,
+  id: PropTypes.string.isRequired,
+  value: PropTypes.string.isRequired,
+  disabled: PropTypes.string,
```

Comment:
Did you intend for this to be a string? Maybe a boolean would be better here then you could do `disabled={disabled ? 'disabled' : ''}` or something similar

Reply:
Just added Boolean casting to TextField:
```
<TextField
          label="SYMBOL"
          id="name"
          disabled={Boolean(securityId)}
          value={state.symbol}
          onChange={event => dispatch(action.setSymbol(event.target.value))}
        />
```